### PR TITLE
Add sync task in gulp

### PR DESCRIPTION
**Pull request by qinezh**

### CODE REVIEW

File: tools/Deployment/gulpfile.js [hunk 1]
```diff
@@ -185,6 +187,20 @@ gulp.task("publish:chocolatey", () => {
     return Chocolatey.publishToChocolateyAsync(releaseNotePath, assetZipPath, chocoScript, nuspec, homeDir, chocoToken);
 });
 
+gulp.task("sync", () => {
```
Comment (vicancy) [inline tools/Deployment/gulpfile.js]: naming: "sync" sounds too general...

Comment (qinezh) [inline tools/Deployment/gulpfile.js]: How about `syncBranch`?

Comment (vicancy) [inline tools/Deployment/gulpfile.js]: sounds good

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**